services/users/utils.go

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package users
+
+import (
+	"strings"
+
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/utils"
+)
+
+func (us *UserService) IsPasswordValid(password string) *model.AppError {
+
+	if *us.config().ServiceSettings.EnableDeveloper {
+		return nil
+	}
+
+	return utils.IsPasswordValidWithSettings(password, &us.config().PasswordSettings)
+}
+
+// CheckUserDomain checks that a user's email domain matches a list of space-delimited domains as a string.
+func CheckUserDomain(user *model.User, domains string) bool {
```

Comment:
Should these be exported?

Reply:
Not for the `users` package, good point. 👍